### DISCUSSION TITLE
Rencontre enregistrée

### DISCUSSION THREAD

**Discussion by CollRen** [582-41E-MA/projet-web-2-CollRen#113] [Remises]
@NabilDjerroud @claraquintela 
Bonjour à vous deux. Je suis entrain de faire le travail sur la rencontre enregistrée; êtes-vous au cours ce matin? Je pourrais vous rejoindre en virtuel...

À plus tard,

**Comment by NabilDjerroud** [comment]
@CollRen Peut on faire une petite rencontre s'il te plait  ?
